modules/mongodb/charts.go

```diff
@@ -432,3 +441,105 @@ var (
 		},
 	}
 )
+
+var (
+	chartDBStatsCollections = &module.Chart{
+		ID:    "database_collections",
+		Title: "Collections",
+		Units: "collections",
+		Fam:   "database_statistics",
+		Ctx:   "mongodb.database_collections",
+		Type:  module.Stacked,
+	}
+
+	chartDBStatsIndexes = &module.Chart{
+		ID:    "database_indexes",
+		Title: "Indexes",
+		Units: "indexes",
+		Fam:   "database_statistics",
+		Ctx:   "mongodb.database_indexes",
+		Type:  module.Stacked,
+	}
+
+	chartDBStatsViews = &module.Chart{
+		ID:    "database_views",
+		Title: "Views",
+		Units: "views",
+		Fam:   "database_statistics",
+		Ctx:   "mongodb.database_views",
+		Type:  module.Stacked,
+	}
+
+	chartDBStatsDocuments = &module.Chart{
+		ID:    "database_documents",
+		Title: "Documents",
+		Units: "documents",
+		Fam:   "database_statistics",
+		Ctx:   "mongodb.database_documents",
+		Type:  module.Stacked,
+	}
+
+	chartDBStatsSize = &module.Chart{
+		ID:    "database_storage_size",
+		Title: "Disk Size",
+		Units: "bytes",
+		Fam:   "database_statistics",
+		Ctx:   "mongodb.database_storage_size",
+		Type:  module.Stacked,
+	}
+)
+
+// updateDBStatsCharts adds dimensions for new databases and
+// removes for dropped
+func (m *Mongo) updateDBStatsCharts(databases []string) {
+	// remove dims for not existing databases
+	m.removeObsoleteDims(databases)
+
+	// add dimensions for new databases
+	for _, database := range sliceDiff(databases, m.discoveredDBs) {
+		for _, chart := range *m.chartsDbStats {
+			id := chart.ID + "_" + database
+			err := chart.AddDim(&module.Dim{ID: id, Name: database, Algo: module.Absolute})
+			if err != nil {
+				m.Errorf("failed to add dim: %s, %s", id, err)
+				continue
+			}
+			chart.MarkNotCreated()
+		}
+	}
+
+	// update the cache
+	m.discoveredDBs = databases
+}
+
+// removeObsoleteDims removes dimensions from dbstats
+// charts for dropped databases
+func (m *Mongo) removeObsoleteDims(newDatabases []string) {
+	diff := sliceDiff(m.discoveredDBs, newDatabases)
+	for _, name := range diff {
+		for _, chart := range *m.chartsDbStats {
+			id := chart.ID + "_" + name
+			err := chart.MarkDimRemove(id, true)
+			if err != nil {
+				m.Warningf("failed to remove dimension %s with error: %s", id, err.Error())
```

Comment:
```suggestion
			if err != nil {
				m.Warningf("failed to remove dimension %s with error: %v", id, err)
```

"Error()" is called for both "v" and "s" verbs

Reply:
done